drivers/virtualbox/network.go

```diff
@@ -158,12 +158,7 @@ func getHostOnlyAdapter(nets map[string]*hostOnlyNetwork, hostIP net.IP, netmask
 	return nil
 }
 
-func getOrCreateHostOnlyNetwork(hostIP net.IP, netmask net.IPMask, vbox VBoxManager) (*hostOnlyNetwork, error) {
-	nets, err := listHostOnlyAdapters(vbox)
-	if err != nil {
-		return nil, err
-	}
-
```

Comment:
moved listHostOnlyAdapters() up one level to prevent VBoxManager from being called twice in a row with the same command.